src/Microsoft.Data/StringDataFrameColumn.BinaryOperations.cs

```diff
@@ -26,22 +26,36 @@ public override DataFrameColumn Add(DataFrameColumn column, bool inPlace = false
             return ret;
         }
 
-        public override DataFrameColumn Add<T>(T value, bool inPlace = false)
+        public static StringDataFrameColumn operator+(StringDataFrameColumn column, string value)
+        {
+            return column.Add(value);
+        }
+
+        public static StringDataFrameColumn operator+(string value, StringDataFrameColumn column)
+        {
+            return column.Add(value);
```

Comment:
I would think that:

```
"myString" + someColumn
```

Would result in `myString` being prepended to each column value. Here it appears that the order of operations is not important, but to a user I think it is.